hatice-ecevit-2016400138/assignment1/Anagram.java

```diff
@@ -71,12 +73,18 @@ public static boolean areAnagramSentences(String s1,String s2, boolean caseSensi
             boolean[] anagram2 = new boolean[words2.length];
             Arrays.fill(anagram1,Boolean.FALSE);
```

Comment:
Actually, Boolean arrays in Java are defined to be filled with "FALSE"  to begin with, so there is actually no need to explicitly initialize them.